hud-data/thughud.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ThugHud",
+  "author": "Russ",
+  "contributors": [
+    "N1te (Sex Hud creator)"
```

Comment:
Just have their name, not what they created

Reply:
done